plasmapy/formulary/tests/test_transport.py

```diff
@@ -55,6 +55,8 @@ class Test_classical_transport:
     @classmethod
     def setup_class(self):
         """set up some initial values for tests"""
+        # TODO: when implementing validate_quantities remove this set equivalencies...
+        #       it affects the entire test environment
```

Comment:
Oh. Yeah, it probably would... It would probably be best to apply equivalencies to tests via context managers, [as per Astropy docs](https://docs.astropy.org/en/stable/units/equivalencies.html#using-equivalencies-in-larger-pieces-of-code), but I'd expect a lot of code to break if we removed this here. Definitely material for another issue...

Reply:
This is now #725 !

Reply:
A context manager could be a useful thing for the future. I’ve done a with decorator using `h5py.File` that worked beautifully. 

Luckily, this bit of code is not necessary when `validate_quantities` is properly applied. I’ve already deleted this TODO and the referenced code in PR #722 .